astropy/utils/masked/tests/test_functions.py

```diff
@@ -54,7 +54,12 @@ def test_ufunc_inplace_no_masked_input(self):
 
     def test_ufunc_inplace_error(self):
         out = np.zeros(self.ma.shape)
-        with pytest.raises(TypeError):
```

Comment:
This is not correct: it is only OK to put something in a mask-less `out` if there are no masks involved.

Reply:
Removed from this PR.